content/contact/_index.md

```diff
@@ -7,7 +7,7 @@ title = "Contact Us"
 The GNU Radio project can be reached
 
 * via Email using the GNU Radio discussion mailing list `discuss-gnuradio@gnu.org`, to which you can sign up [here](https://lists.gnu.org/mailman/listinfo/discuss-gnuradio),
-* via Matrix on our Matrix homeserver, e.g. using [chat.gnuradio.org](https://chat.gnuradio.org), or
+* via Matrix on our Matrix homeserver, e.g. using [chat.gnuradio.org](https://matrix.to/#/#gnuradio-community:gnuradio.org), or
```

Comment:
Now the link text says "using chat.gnuradio.org", which means "using the element client hosted on chat.gnuradio.org", but the link target is just a generic matrix.to link, bringing users through complicated client choice instead of the chat.gnuradio.org client. (we really want everyone to join as easily as possible, agreed, but the matrix.to "oh here's a bunch of clients you might choose from" is really counter-productive there, if a user has IT problems already. We do have existing user feedback on that.)

So, don't think that's what we intended here, I'm afraid.

Reply:
Oh. Do you think that having a "hybrid" solution could work? (like, a signup (joining on chat.gnuradio.org) or join link (joining to matrix.to)) ?

Or just beeing more verbose in the line that speaks about matrix?